tests/containers/image.pm

```diff
@@ -61,7 +84,7 @@ sub run {
         record_info "IMAGE", "Testing image: $image Version: $version";
         test_container_image(image => $image, runtime => $engine);
         $self->test_rpm_db_backend(image => $image, runtime => $engine);
-        test_systemd_install(image => $image, runtime => $engine);
+        $self->test_systemd_install(image => $image, runtime => $engine);
```

Comment:
Does it need `self`?

Reply:
> Does it need `self`?

I wouldn't bet my life on it. Perl's OOP is weird to me.